src/gui/touchscreengui.cpp

```diff
@@ -110,7 +111,15 @@ static EKEY_CODE id_to_keycode(touch_gui_button_id id)
 			break;
 	}
 	assert(!key.empty());
-	return keyname_to_keycode(g_settings->get("keymap_" + key).c_str());
+	std::string resolved = g_settings->get("keymap_" + key);
+	try {
+		code = keyname_to_keycode(resolved.c_str());
+	} catch (UnknownKeycode &e) {
+		code = KEY_KEY_CODES_COUNT;
```

Comment:
I think `KEY_UNKNOWN` would fit better

Reply:
oh lol, we discussed `KEY_NONE` (can't be used because it's a real key) vs. `KEY_KEY_CODES_COUNT`, but looks like both of us didn't see `KEY_UNKNOWN`

https://github.com/minetest/minetest/pull/14075#discussion_r1439275487